x-pack/plugins/ml/public/jobs/new_job/simple/components/enable_model_plot_checkbox/enable_model_plot_checkbox_directive.js

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { EnableModelPlotCheckbox } from './enable_model_plot_checkbox_view.js';
+import { ml } from 'plugins/ml/services/ml_api_service';
+
+import { uiModules } from 'ui/modules';
+const module = uiModules.get('apps/ml');
+
+module.directive('mlEnableModelPlotCheckbox', function () {
+  return {
+    restrict: 'AE',
+    replace: false,
+    scope: {
+      formConfig: '=',
+      ui: '=ui',
+      getJobFromConfig: '='
+    },
+    link: function ($scope, $element) {
+      const STATUS = {
+        FAILED: -1,
+        NOT_RUNNING: 0,
+        RUNNING: 1,
+        FINISHED: 2,
+        WARNING: 3,
+      };
+      const errorHandler = (error) => {
```

Comment:
nit, there are three different function definition formats in this file:
Arrow: `const errorHandler = (error) => {`
Expression: `const validateCardinality = function () {`
Declaration: `function updateCheckbox() {`

Unless the function is in the `$scope`, we should use function declarations. Kibana also mentions this in their style guide.
https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#function-definitions